Slipe/Core/Source/SlipeClient/Rendering/Renderer.cs

```diff
@@ -5,32 +5,23 @@
 using Slipe.MtaDefinitions;
 using Slipe.Shared.Elements;
 using Slipe.Client.Elements;
+using Slipe.Client.Rendering.Events;
 
 namespace Slipe.Client.Rendering
 {
     /// <summary>
     /// Singleton responsible for rendering
```

Comment:
Doesn't seem to be a singleton or sstill responsible for rendering stuff anymore.